controllers/dscinitialization/utils.go

```diff
@@ -101,6 +103,14 @@ func (r *DSCInitializationReconciler) createOdhNamespace(ctx context.Context, ds
 				r.Log.Error(err, "Unable to fetch monitoring namespace", "name", monitoringName)
 				return err
 			}
+		} else { // force to patch monitoring namespace with label for cluster-monitoring
+			r.Log.Info("Patching monitoring namespace for Managed cluster", "name", monitoringName)
```

Comment:
Wouldn't this add the namespace label https://github.com/red-hat-data-services/rhods-operator/pull/98/files#diff-5f4497af58ef4b32b811ace8e826c31377aa6ddd140b859e88dd84b876e7139aR93?